soh/src/code/z_demo.c

```diff
@@ -2073,29 +2090,43 @@ void Cutscene_HandleConditionalTriggers(GlobalContext* globalCtx) {
 
     if ((gSaveContext.gameMode == 0) && (gSaveContext.respawnFlag <= 0) && (gSaveContext.cutsceneIndex < 0xFFF0)) {
```

Comment:
given that almost all branches in here only do something `if (!gSaveContext.n64ddFlag)` I think we can tidy up a bit

Reply:
i agree, but i'm not quite sure what the best way to do it would be considering the ganon's tower part at the end

just moving the extra `if(!gSaveContext.n64ddFlag)` logic into the main conditionals would do a bit of cleanup work, but i feel like there might be a better way.

any suggestions?

Reply:
It's tricky I see. let's at least move the `!gSaveContext.n64ddFlag` check to the outer ifs?
As far as I can tell from the logic that should not adversely affect things since they're all very explicit checks.